simul/platform/runsimul.go

```diff
@@ -99,6 +105,10 @@ func Simulate(serverAddress, simul, monitorAddress string) error {
 			timeout *= 2
 		}
 		childrenWait.Record()
+		log.Lvl2("Broadcasting start")
+		for _, conode := range rootSC.Tree.Roster.List {
+			rootSC.Server.Send(conode, &startSimulation{})
+		}
```

Comment:
Good question - I don't think it's necessary. This is the root-node that contacts all other nodes (including itself). What happens if one conode is still in `simulation.Node` and then the root-node calls its `simulation.Root`? Is it bad?